### ISSUE TITLE
Linting occurs, but the results of linting don't display

### ISSUE THREAD

**Issue by skylineproject** [AtomLinter/linter-javac#12]
Hi all, thanks for your work on this!

Running the current version of Atom (0.176), linter, and linter-javac, it seems that errors found are not being reported back to the console.

Here's the output to the console after one save:
```javascript
stderr 1 error

javac returned LinterJavac {editor: Object, cwd: "/{Path to current working directory}", subscriptions: CompositeDisposable, executionTimeout: 5000, executablePath: "/usr/bin/javac"…}cwd: "{Path to current working directory}"editor: ObjectexecutablePath: "/usr/bin/javac"executionTimeout: 5000subscriptions: CompositeDisposable__proto__: ctor []
```

Even with the indication that an error occurred, the error count in the lower-right corner or Atom doesn't increase:

![screen shot 2015-01-28 at 8 05 28 pm](https://cloud.githubusercontent.com/assets/5009403/5951186/33372c08-a729-11e4-8e80-1553eb73de8f.png)

I've used other linters (linter-jshint for JavaScript, primarily) and they seem to work fine (when errors are present, errors show up in that counter shown in the image above). I have the executable path for javac entered in config.cson (as you can see in the log above).

Thanks for any assistance you can provide, and for the work you've done to make linter so awesome!

Brandon
(MacBook Air, OS X 10.10.2)

**Comment by skylineproject** [comment]
It turns out this is related to an issue with the regex used. I have prepared a PR (#13) to mitigate this.
